webapp/src/components/content/attachmentElement.tsx

```diff
@@ -122,8 +122,18 @@ const AttachmentElement = (props: Props): JSX.Element|null => {
         document.body.removeChild(anchor)
     }
 
+    const handleMouseLeave = () => {
```

Comment:
Hmm, DOM manipulation is an expensive operation. I suppose you're doing this to remove focus from something? There must be a better way for this.